### PR TITLE
Julia Documentation

### PR DESCRIPTION
**Pull request by michel2323**
This is a first stab at writing documentation for running Julia on Polaris.

Polaris is currently down. I'll do another pass these days and check whether the scripts work. Any feedback you can give me is welcome.

As a general note, I am not sure Julia fits nicely under the "data science" section. We certainly haven't used it that way.

Closes (partially) #220

### CODE REVIEW

**Comment by felker** [comment]
Thanks again for doing this. Are you going to include a ThetaGPU counterpart in this PR too?

Regarding the "data science" sidebar section, this is related to the discussion in #219 . We can start a new folder called "languages" in this PR and put your Julia documentation there. I will move Python over at some point.

File: docs/polaris/data-science-workflows/julia.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+The following example shows how to use MPI.jl, CUDA.jl, and HDF5.jl to write a
+parallel program that computes the sum of two vectors on the GPU and writes the
+result to an HDF5 file. A repository with the example code computing an
+approximation of pi can be found at
+[Polaris.jl](https://github.com/exanauts/Polaris.jl). In this repository, you will also find
+a `setup_polaris.sh` script that will build the HDF5.jl and MPI.jl package for the system libraries.
+The dependencies can be installed with the following commands:
+```bash
+julia --project
+```julia
```
Comment (felker) [inline docs/polaris/data-science-workflows/julia.md:100]: This part is rendered weird in GitHub Flavored Markdown for me--- does it look normal after building with MkDocs? Seems like it is missing closing ``` for each of the 3x code blocks

**Comment by michel2323** [comment]
> Thanks again for doing this. Are you going to include a ThetaGPU counterpart in this PR too?

We are not using ThetaGPU anymore. Only if it is useful to someone, otherwise I wouldn't mind skipping it.

File: docs/polaris/data-science-workflows/julia.md (hunk 1 above)
Comment (michel2323) [reply docs/polaris/data-science-workflows/julia.md:102]: Fixed.

**Review by felker — APPROVED** [review]

**Pull request merged**